Maestro/subscriptions.json

```diff
@@ -156,6 +172,43 @@
       ]
     },
     {
+      "path": "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectk-tfs/release/1.0.0/Latest.txt",
+      "handlers": [
+        // This handler will bring the Latest ProjectK TFS release/1.0.0 build into CoreFX release/1.0.0
+        {
+          "maestroAction": "corefx-general",
+          "maestroDelay": "00:10:00",
+          "vsoSourceBranch": "release/1.0.0",
+          "ScriptFileName": "UpdateDependencies.ps1",
+          "Arguments": [
+            "-GitHubUser dotnet-bot",
+            "-GitHubEmail dotnet-bot@microsoft.com",
+            "-GitHubPassword `$(`$Secrets['DotNetBotGitHubPassword'])",
+            "-GitHubUpstreamBranch release/1.0.0",
+            "-VersionFileUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/projectk-tfs/release/1.0.0/Latest.txt",
+            "-GitHubPullRequestNotifications 'dotnet/corefx-contrib'",
+            "-DirPropsVersionElements ExternalExpectedPrerelease"
+          ]
+        },
+        // This handler will bring the Latest ProjectK TFS release/1.0.0 build into CoreCLR release/1.0.0
```

Comment:
`projectk-tfs/release/1.0.0` is ProjectKRel. The "ProjectK" is meant to refer to the project without any branch suffix. I'll clarify the comment.